networking/networking.go

```diff
@@ -83,8 +84,18 @@ func Setup(podRoot string, podID types.UUID, fps []commonnet.ForwardedPort, netL
 		},
 	}
 
+	err := os.MkdirAll(mountNetnsDirectory, 0755)
+	if err != nil {
+		return nil, err
+	}
+
+	err = n.mountNetnsDirectory(false)
```

Comment:
The container will start well but we cannot guarantee the lifecycle of its network if a third-party software like `ip netns add` is triggered after its starts. So I'm definitely for a hard-fail.